parlai/tasks/wikipedia/agents.py

```diff
@@ -51,7 +51,7 @@ def setup_data(self, path):
                         if self.key_value:
                             yield (title, [text]), True
                         else:
-                            yield (title + '\n' + text, None), True
+                            yield (text, None), True
```

Comment:
no more title?

Reply:
The title is actually in the `text` variable; this was adding the title twice to the example